app/models/child.rb

```diff
@@ -3,19 +3,23 @@
 # A child in care at businesses who need subsidy assistance
 class Child < UuidApplicationRecord
   belongs_to :user
+  has_many :child_sites, dependent: :destroy
+  has_many :sites, through: :child_sites
 
   validates :active, inclusion: { in: [true, false] }
   validates :date_of_birth, presence: true
   validates :full_name, presence: true
   validates :full_name, uniqueness: { scope: %i[date_of_birth user_id] }
 
   validates_each :date_of_birth do |record, attr, value|
```

Comment:
Date validation is being done in multiple models with the same code.
I'd suggest creating a custom validator class and then using it in all the places.